### ISSUE TITLE
Support of PubMed format

### ISSUE THREAD

**Issue by mouradvt04** [rayyanqcri/rayyan-formats-plugins#21]
PubMed has a new interface and  is still keeping the legacy PubMed which may be discontinued at any time. We need to make sure that our users can get data from PubMed to Rayyan. 
PubMed created their own format in their new interface, called PubMed (has some similarities with the RIS format)!
For reference, see the below communication with US NLM support

Hi Dr. Ouzzani, 

Thank you for writing to the help desk.  We do not anticipate adding XML to the new PubMed web interface. PubMed data is available in XML format through our E-utilities APIs (https://www.ncbi.nlm.nih.gov/books/NBK25497/) and our Data Distribution program (https://pubmed.ncbi.nlm.nih.gov/help/#download-pubmed-data).

An example EFetch URL that retrieves XML data for 2 PMIDs: https://eutils.ncbi.nlm.nih.gov/entrez/eutils/efetch.fcgi?db=pubmed&id=11748933,11700088&retmode=xml

A new API will be developed for PubMed; however, we do not yet have a timeline to share regarding the update. Please subscribe to the E-Utilities Announcement Mailing List: https://www.ncbi.nlm.nih.gov/mailman/listinfo/utilities-announce/

From the Save menu, you can also use the "PubMed" format which has replaced RIS.  This file will include the abstracts.

Kind regards, 
Marie Collins
U.S. National Library of Medicine
****
* PLEASE DO NOT MODIFY THE SUBJECT LINE OF THIS EMAIL WHEN RESPONDING TO ENSURE CORRECT TRACKING *

Case Information:
Case #: CAS-514295-C2Q7Z7
Customer Name: Mourad OUZZANI
Customer Email: mouzzani@hbku.edu.qa
Case Created: 2020-04-01T12:49:27Z

Summary: Saving results

Details: Hi,

In the legacy PubMed, one can save results using your XML format which was handy for getting all the bibliographic information including asbtracts and using them in other systems (such as on rayyan.qcri.org) then in the new PubMED I've used the RIS format few times but now it disappeared!
It will be great to have a standard reference format that would include abstracts such as RIS or CIW or simply the CSV format. The previous XML format would work too.

Best,
Mourad Ouzzani, PhD

**Comment by mouradvt04** [comment]
This is the message from PubMed/NLM:
"All traffic will be directed to the new PubMed on or after May 18th.  That is the date that you should plan to use the new PubMed, and no longer use the legacy system."
So we need to work on the new parser asap.

**Comment by hammady** [comment]
Why is this urgent? We should disable the pubmed source instead.

**Comment by mouradvt04** [comment]
Rayyan users won't be able to bring in searches from PubMed "on or after May 18th". It's possible it stays longer though. 

So we should disable the topic search (which should be easy to do), then write a parser for the new PubMed format, and finally fix the topic search (although I don't see the topic search useful except to quickly test how Rayyan works as it's rate-limited).

**Comment by mouradvt04** [comment]
Of interest to this issue from PubMed FAQ on their new interface:
Why isn't RIS format an option on the Save menu?
Based on feedback from our users, we have restored the legacy MEDLINE format (now called PubMed format in the new site) for Save and Cite, and removed the RIS format download.

In the short time we offered the RIS format in the new PubMed, we heard from a number of users requesting extensive modifications, as it appears that RIS is used in different ways by various external resources. For example, RIS has several fields for journal title, and we received conflicting feedback about which fields to use and what data to provide in those fields. 

At this time, it is not feasible to maintain multiple formats for the same purpose of exporting to a citation manager. Based on the user feedback we have collected, we are restoring the legacy MEDLINE field tag format, renamed PubMed format, to meet users' needs for the complete PubMed record as well as a file that can be used with external citation management software.
